lms/djangoapps/certificates/migrations/0015_auto_20190404_1514.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Nit: It would be useful to give this migration a more descriptive name than the default.